WolfpackBot.Data/Migrations/20211105183534_ChangeStandingsMessageIdToList.cs

```diff
@@ -0,0 +1,33 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace WolfpackBot.Data.Migrations
+{
+    public partial class ChangeStandingsMessageIdToList : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.DropColumn(
```

Comment:
Maybe this does some magic stuff in the background but wont this lose the data in that column?

Reply:
It's OK at this point because there won't be any data in this column. We haven't got any standings messages posted to have their IDs